src/sentry/static/sentry/less/sentry-hovercard.less

```diff
@@ -67,7 +67,7 @@
 
   h6 {
     color: @50 !important;
-    font-size: 11px;
+    font-size: 11px !important;
```

Comment:
🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔

Reply:
Just hovercard things :(

We need this because we don't get the CSS that overrides this if we're outside of the Releases sidebar section. But really, hovercards should render at the body level and be positioned using Popper.